### PR TITLE
docs: refactor to use a cli only approach

### PR DESCRIPTION
**Pull request by Nebulis**

### CODE REVIEW

File: docs/appendix/glossary.md [hunk 1]
```diff
@@ -22,6 +22,10 @@ Placeholder
 
 Placeholder
 
+### Wallet
+
+Placeholder
+
```
Comment (simboonlong) [inline docs/appendix/glossary.md:28]: glossary is useful, any plans to update the placeholders?

File: website/sidebars.json [hunk 2]
```diff
@@ -27,8 +28,16 @@
     "Advanced": [
       "extension/overview",
       "extension/verification-methods",
-      "extension/identity-proofs"
+      "extension/identity-proofs",
+      "extension/custom-renderer",
+      "extension/configuring-dns"
     ],
```
Comment (simboonlong) [inline website/sidebars.json:34]: wanna either stick to `Advanced` + `advanced/overview` etc or `Extension`+ `extension/overview` etc?

File: docs/verifiable-document/dns-proof.md [hunk 3]
```diff
@@ -8,28 +8,17 @@ Every OA document's provenance can be verified and traced back to its creator or
...
-The OpenAttestation CLI is a good solution when starting with OpenAttestation as it will help you to quickly create a short-lived DNS TXT record. If you want to use a custom domain, you should head to [using a custom domain](#using-a-custom-domain) section
-
-### Prerequisites
-
-- [OpenAttestation CLI](/docs/component/open-attestation-cli) installed
-- A document store (for instance created from the [previous guide](./document-store))
+In this example above, the document's issuer is bound to `demo.openattestation.com`.
 
-### Creating the DNS TXT record
+With your [document store](/docs/verifiable-document/document-store)), run the following command:
```
Comment (simboonlong) [inline docs/verifiable-document/dns-proof.md:15]: typo double ))

File: docs/appendix/glossary.md (hunk 1 above)
Comment (Nebulis) [reply docs/appendix/glossary.md:28]: not in this MR but yeah

File: website/sidebars.json (hunk 2 above)
Comment (Nebulis) [reply website/sidebars.json:34]: I will change to advanced/xxx

Comment (Nebulis) [reply website/sidebars.json]: changed

File: docs/verifiable-document/dns-proof.md (hunk 3 above)
Comment (Nebulis) [reply docs/verifiable-document/dns-proof.md]: changed

File: docs/advanced/configuring-dns.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
+---
```
Comment (Nebulis) [inline docs/advanced/configuring-dns.md:1]: I need to review opencerts docs in case there is any broken links :)

**Pull request merged**